composer.json

```diff
@@ -18,7 +18,8 @@
     "require-dev": {
         "webmozart/key-value-store": "^1.0.0-beta5",
         "symfony/filesystem": "^2.0",
-        "phpunit/phpunit": "^4.6"
+        "phpunit/phpunit": "^4.6",
+        "sebastianbergmann/version": "^1.0.1"
```

Comment:
sorry, the composer package name is actually ``sebastian/version`` (the namespace for composer does not match the github username)